.github/workflows/deploy.yml

```diff
@@ -0,0 +1,134 @@
+name: deploy
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+
+  create-windows-binaries:
+
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install stable
+      uses: dtolnay/rust-toolchain@stable
+      with:
+        toolchain: stable
+
+    - name: Build cargo-careful
+      run: |
+        cargo build --release
+
+    - name: Get the version
+      shell: bash
+      id: tagName
+      run: |
+        VERSION=$(cargo pkgid | cut -d# -f2)
+        echo "tag=$VERSION" >> $GITHUB_OUTPUT
+
+    - name: Build package
+      id: package
+      shell: bash
+      run: |
+        echo "file=./target/release/cargo-careful.exe" >> $GITHUB_OUTPUT
+        echo "name=cargo-careful.exe" >> $GITHUB_OUTPUT
+
+    - name: Upload artifacts
+      uses: actions/upload-artifact@v2
+      with:
+        name: ${{ steps.package.outputs.name }}
+        path: ${{ steps.package.outputs.file }}
+
+  create-unix-binaries:
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        include:
+          - os: ubuntu-latest
+            target: x86_64-unknown-linux-musl
+          - os: macos-latest
+            target: x86_64-apple-darwin
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install Rust stable
+      uses: dtolnay/rust-toolchain@stable
+      with:
+        toolchain: stable
+        target: ${{ matrix.target }}
+
+    - name: Install musl
+      if: contains(matrix.target, 'linux-musl')
+      run: |
+        sudo apt-get install musl-tools
+
+    - name: Build cargo-careful
+      run: |
+        # TODO: Remember to add RUSTFLAGS=+crt-static for musl target when
+        # static linkage will not be the default behaviour
```

Comment:
How would I even do that? Sounds quite tedious to do that with GHA.

Reply:
Since this action has been automatically generated from a tool, we can remove this comment because it's just a reminder

Reply:
Are you sure of this? It sounds like in the future this will default to linking the CRT dynamically which would be an issue for distributable binaries.

Reply:
Yep, better to be explicit, so link CRT statically